packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionStart/ForceFunctionStartExecutor.ts

```diff
@@ -81,7 +81,8 @@ export abstract class ForceFunctionStartExecutor extends LibraryCommandletExecut
           return new Promise(resolve =>
             resolve(this.cancelFunction(registeredStartedFunctionDisposable))
           );
-        }
+        },
+        isContainerLess: false
```

Comment:
Can you update the unit tests for the three functionStartExecutor classes?

Reply:
I looked but didn't see any places but didn't find any code (in out tests files) where they needed to be updated.  If there's something specific you have in mind, ping me & lets go over what needs to be updated.

Reply:
done